appcli/variables_manager.py

```diff
@@ -28,13 +31,15 @@
 class VariablesManager:
     """Manages the configuration variables"""
 
-    def __init__(self, configuration_file):
+    def __init__(self, configuration_file, extra_configuration_files):
         """Creates a manager for the specified file
 
         Args:
-            configuration_file (str): Path to the configuration file to manage
+            configuration_file (str): Path to the configuration file to manage.
+            extra_configuration_files (str): Path to a directory containing extra configuration files.
         """
         self.configuration_file = Path(configuration_file)
+        self.extra_configuration_files = Path(extra_configuration_files)
```

Comment:
Same as above, should probably rename to show it's a directory containing extra configuration files

Reply:
Fixed.